src/session/session_api.c

```diff
@@ -521,37 +538,42 @@ __session_reconfigure(WT_SESSION *wt_session, const char *config)
         else
             F_CLR(session, WT_SESSION_DEBUG_RELEASE_EVICT);
     }
+    WT_RET_NOTFOUND_OK(ret);
 
-    WT_ERR_NOTFOUND_OK(ret, false);
-
-    ret = __wt_config_getones(session, config, "cache_max_wait_ms", &cval);
-    if (ret == 0 && cval.val)
+    if ((ret = __wt_config_getones(session, config, "cache_max_wait_ms", &cval)) == 0)
         session->cache_max_wait_us = (uint64_t)(cval.val * WT_THOUSAND);
-    WT_ERR_NOTFOUND_OK(ret, false);
+    WT_RET_NOTFOUND_OK(ret);
 
-    if (S2C(session)->prefetch_auto_on)
-        F_SET(session, WT_SESSION_PREFETCH_ENABLED);
-    else
-        F_CLR(session, WT_SESSION_PREFETCH_ENABLED);
+    return (0);
+}
+
+/*
+ * __session_reconfigure --
+ *     WT_SESSION->reconfigure method.
+ */
+static int
+__session_reconfigure(WT_SESSION *wt_session, const char *config)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    session = (WT_SESSION_IMPL *)wt_session;
+    SESSION_API_CALL_PREPARE_NOT_ALLOWED(session, ret, reconfigure, config, cfg);
+    WT_UNUSED(cfg);
```

Comment:
As discussed, this can be removed.

Reply:
Removed the line in [de629ca](https://github.com/wiredtiger/wiredtiger/pull/11177/commits/de629caa8874c04ba4ff4c8b305fa3a308a43395)